worldengine/cli/main.py

```diff
@@ -441,10 +450,10 @@ def main():
     if args.temps and len(args.temps.split('/')) is not 6:
         usage(error="temps must have exactly 6 values")
 
-    temps =[.874, .765, .594, .439, .366, .124]
+    temps = [.874, .765, .594, .439, .366, .124]
     if args.temps:
         temps = args.temps.split('/')
-        for x in range(0,6):
+        for x in range(0,len(temps)):
```

Comment:
mmm I see. I would in that case issue a warning, so that users who invert the order of items by mistake could realize it. If then they would like to do so... free to do it